src/Integration/MefServices/ActiveSolutionTracker.cs

```diff
@@ -86,8 +89,7 @@ int IVsSolutionEvents.OnBeforeUnloadProject(IVsHierarchy pRealHierarchy, IVsHier
 
         int IVsSolutionEvents.OnAfterOpenSolution(object pUnkReserved, int fNewSolution)
```

Comment:
This event was triggered when opening the solution not when the solution was fully loaded.